torch/_dynamo/output_graph.py

```diff
@@ -477,6 +477,9 @@ def __init__(
         )
 
         self.guard_on_key_order: set[str] = set()
+        self.saved_tensors_hooks_subgraph_names: Optional[list[str]] = (
+            self.maybe_install_saved_tensors_hooks_subgraphs()
```

Comment:
can you sanity check that in tlparse, the dynamo graph output includes these pack/unpack graphs?